R/build-home-index.R

```diff
@@ -39,17 +45,50 @@ data_home <- function(pkg = ".") {
 
 data_home_sidebar <- function(pkg = ".") {
   pkg <- as_pkgdown(pkg)
-  if (!is.null(pkg$meta$home$sidebar))
+  if (isFALSE(pkg$meta$home$sidebar))
     return(pkg$meta$home$sidebar)
 
+  sidebar_path <- file.path(pkg$src_path, pkg$meta$home$sidebar$html)
```

Comment:
If this is a pointer to a path, it should be `html_path` or similar. I wonder if this should be in addition to the regular sidebar, rather than a replacement?

Reply:
I'm not sure.
* If you want a really custom sidebar you'd provide your own HTML as sidebar.html field.
* If you want to add a custom section (H2 title, any HTML below that), you'd use the sidebar.structure and sidebar.components fields.

Reply:
Do we have evidence that folks want to replace the whole sidebar?

Reply:
no but this PR already provides a way to add custom component to the navbar.